packages/api/src/logFormatter/bin.ts

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env node
+import split from 'split2'
+
+import { LogFormatter } from '.'
```

Comment:
```suggestion
import { LogFormatter } from './index'
```
I know dot is ok, but this is the convention we've been using